pkg/boot/universalpayload/utilities.go

```diff
@@ -174,6 +174,52 @@ type FbFixScreenInfo struct {
 	Reserved     [2]uint16
 }
 
+type MCFGBaseAddressAllocation struct {
+	BaseAddr  EFIPhysicalAddress
+	PciSegGrp uint16
+	StartBus  uint8
+	EndBus    uint8
+	Reserved  uint32
+}
+
+type ResourceInfo struct {
+	Type        string
+	BaseAddress string
+	EndAddress  string
+	Attributes  string
+}
+
+type ResourceRegions struct {
+	Mmio64Base  uint64
```

Comment:
probably it should be MMIO64Base and so on. https://go.dev/wiki/CodeReviewComments#initialisms